QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,195 @@
+This document is a declaration of software quality for the `libstatistics_collector` package, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
+
+# `libstatistics_collector` Quality Declaration
+
+The package `libstatistics_collector` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Quality Categories in REP-2004](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-quality-categories) of the ROS2 developer guide.
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`libstatistics_collector` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning)
+
+### Version Stability [1.ii]
+
+`libstatistics_collector` is at a stable version (`>= 1.0.0`.)
+The current version can be found in its [package.xml](package.xml), and its change history can be found in its [CHANGELOG](CHANGELOG.rst).
+
+### Public API Declaration [1.iii]
+
+All symbols in the installed headers are considered part of the public API.
+
+Except for the exclusions listed below, all installed headers are in the `include` directory of the package, headers in any other folders are not installed and considered private.
+
+### API Stability Policy [1.iv]
+
+`libstatistics_collector` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.

```

Comment:
This section is supposed to talk about API stability within major versions right?

Reply:
This still needs to be addressed.